### PR TITLE
spanish localization

### PR DESCRIPTION
**Pull request by WaiYanMHein**
Created a localization for Spanish

### CODE REVIEW

File: localization/italian/event_localization/immortality_set_age/major_decisions_middle_east_l_italian.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+﻿
```
Comment (paige404) [inline localization/italian/event_localization/immortality_set_age/major_decisions_middle_east_l_italian.yml:1]: This one looks unfinished. Were you planning to do Italian in a separate PR? If so, we should remove it from this one.

I'm happy to have both in the same PR, but in that case the Italian localization must also be complete.

**Review by paige404 — CHANGES_REQUESTED** [review]
I don't speak Spanish or Italian so I can't confirm accuracy but this mostly looks good. Just finish up the Italian localization, or remove it from the PR

File: localization/italian/event_localization/immortality_set_age/major_decisions_middle_east_l_italian.yml (hunk 1 above)
Comment (WaiYanMHein) [reply localization/italian/event_localization/immortality_set_age/major_decisions_middle_east_l_italian.yml:1]: ok i will complete the italian localization

Comment (WaiYanMHein) [reply localization/italian/event_localization/immortality_set_age/major_decisions_middle_east_l_italian.yml]: I just finished the italian localization, take a look. : )

**Review by paige404 — APPROVED** [review]
This looks great! Sorry for the delay in review, and thank you for your hard work here.

**Comment by paige404** [comment]
All merged up and uploaded to Steam Workshop. Your work should be live now.
